app/ui/src/app/settings/oauth-apps/oauth-app-form.component.ts

```diff
@@ -106,14 +72,13 @@ export class OAuthAppFormComponent implements OnInit {
   }
 
   ngOnInit() {
-    const formConfig = (this.formConfig = JSON.parse(
-      JSON.stringify(OAUTH_APP_FORM_CONFIG)
-    ));
-    this.formModel = this.formFactory.createFormModel(
-      formConfig,
-      this.item.client,
-      ['*']
-    );
+    this.store.list.subscribe(oauthApps => {
```

Comment:
I would store the Subscription return value from this and have the component also implement `OnDestroy`, then add an `ngOnDestroy()` function and unsubscribe, like this -> https://github.com/syndesisio/syndesis/blob/master/app/ui/src/app/integration/edit-page/action-configure/action-configure.component.ts#L325-L327